cli/azd/cmd/init.go

```diff
@@ -201,7 +201,7 @@ func (i *initAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 		tracing.SetUsageAttributes(fields.InitMethod.String("app"))
 
 		header = "Your app is ready for the cloud!"
-		followUp = "You can provision and deploy your app to Azure by running the " + color.BlueString("azd up") +
```

Comment:
Was there a reason we were directly using blue string here instead of our output wrappers?

Reply:
Indeed, they are different colors:
![image](https://github.com/Azure/azure-dev/assets/2322434/46474a1b-74d3-4821-9615-64f666c11d62)

@Austinauth had noticed that we were not using the intended colors. I had made the minimal feature change for simplified init. Let's avoid bundling this change in here since it's a separate functional change if we can. We should have a discussion on what semantic colors we've decided on and add a new category to `output`. I don't know if it's "commands", or something else.

Reply:
Roll back for now.